staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -430,6 +430,12 @@ type ListOptions struct {
 	// compatibility reasons) and to false otherwise.
 	// +optional
 	SendInitialEvents *bool `json:"sendInitialEvents,omitempty" protobuf:"varint,11,opt,name=sendInitialEvents"`
+
+	// SkipCache bypasses the server cache, forcing the request to be served directly from etcd.
+	//
+	// Caution: This dramatically increases the request resource cost.
+	// Use only for debugging cache staleness or corruption. Not intended for regular client usage.
+	SkipCache bool `json:"skipCache,omitempty" protobuf:"varint,12,opt,name=skipCache"`
```

Comment:
should this be *bool so we can distinguish "set to false" from "absent", and reject any setting at all by normal users?

Reply:
+1 for `DebugWatchCacheDirective *string` or similar.  I suspect we'll end up have multiple variations worth having.